src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -112,8 +112,8 @@
   {:path (str "/workspaces/" (id-path workspace-id) "/setAttributes")
    :method :patch})
 
-(defn get-workspace-genomic-operations [workspace-id job-id]
-  {:path (str "/workspaces/" (id-path workspace-id) "/genomics/operations/" job-id)
+(defn get-workflow-genomic-operations [workflow-id job-id]
+  {:path (str "/workflows/v1/" workflow-id "/backend/metadata/" job-id)
```

Comment:
What, if anything, do we need to do to notify the Terra team that this API call has changed?  Or have they not implemented this functionality yet?

Reply:
they have not implemented which is why it needs to be fixed here